SpatialGDK/Documentation/content/how-to-use-schema.md

```diff
@@ -6,45 +6,57 @@
 
 You must generate schema when you add or change any [replicated properties (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties) that you want to deploy to SpatialOS.
 
+> Whenever you generate schema, the GDK automatically creates schema for classes with [replicated properties (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties) or [RPCs (Unreal documentation)](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/RPCs). If a class does not have replicated properties or RPCs, then the GDK does not generate schema for it.
+
 ## How to generate schema
 
-There are two ways to generate schema for your project:
+There are two ways to generate schema for your project from [within the Unreal Editor](#generating-schema-within-the-unreal-editor), and you can also generate schema [via the command line](#generating-schema-via-the-command-line).
+
+### Generating schema within the Unreal Editor
+Within the Unreal Editor, you can generate schema using a Full Scan, or generate schema iteratively. For information on when to use each method, see [Which method to use](#which-method-to-use).
 
-* **Generate schema (Full Scan)** 
+* **Full Scan** 
 
-    To generate schema for all classes* in your project:<br/>
+    To generate schema for all classes in your project:<br/>
```

Comment:
Hmm, I removed the footnote because there were tons of asterisks that all linked to a callout (see https://docs.improbable.io/unreal/alpha/content/how-to-use-schema), and I added the note at the top instead, to avoid the need for a footnote. 

Maybe instead of having a footnote we could just add it inline, so it would be:

> To generate schema for all classes in your project that have replicated properties or RPCs:

What do you reckon?

Reply:
Worried there's an issue for anyone who does a search and lands half-way down the page or scans because it categorically says "To generate schema for all classes" which isn't true.
I'd go for asterisk because it's less intrusive in the text.

Reply:
Have actually gone for adding it in everywhere (as well as having a note about it at the top).

![image](https://user-images.githubusercontent.com/32169118/60514248-035c7680-9cd1-11e9-8eaf-32b0262d8aaa.png)